lib/api/core/models/repositories/profileRepository.js

```diff
@@ -177,9 +180,10 @@ class ProfileRepository extends Repository {
    * Given a Profile object, delete it from memory and database
    *
    * @param {Profile} profile
+   * @param {string} [refresh]
    * @returns {Promise}
    */
-  deleteProfile (profile) {
+  deleteProfile (profile, refresh = '') {
```

Comment:
Nitpicking: instead of a `refresh`as a string argument, I'd rather user `options` as object argument, to be more consistant with `Repository.delete`